src/theme/components/feedbackBox.ts

```diff
@@ -79,6 +80,11 @@ const FeedbackBox = defineMultiStyleConfig({
         zIndex: "10000 !important",
       },
     },
+    radioGroup: {
```

Comment:
This is ok but I think there's a side effect where the gap is reduced between radio options:
<img width="741" alt="Screenshot 2025-05-08 at 3 22 01 PM" src="https://github.com/user-attachments/assets/4e38259f-33ab-439d-a099-af1fbe5264ac" />

Reply:
PR on the left, prod on the right.

Reply:
Good catch. The `layout` prop for the `RadioGroup` component not only adjusts the `flexDirection`. It also adjusts the flex `gap` value (`"s"` vs `"l"`). That will need to also be set here.

Reply:
Yes good catch!